views/index.hbs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Homepage</title>
+  <link rel="stylesheet" href="/styles/styleHome.css">
+</head>
+<body>
+  <div class="search">
```

Comment:
Your form element  doesn't work because the label, input and button need to be nested within it so that you can submit the artist name properly. You should remove the unnecessary div surrounding those elements as well.